### PR TITLE
Update integration test docs [No Jira]

### PR DESCRIPTION
**Pull request by breilly2**
Just some doc updates.

### CODE REVIEW

File: integration-tests/README.md [hunk 1]
```diff
@@ -45,10 +45,13 @@ The following environment variables are parsed by the tests:
 #### Convenient one-liner:
 
 ```sh
+[HEADLESS=false] \
+[ENVIRONMENT=local|dev|alpha|perf|staging] \
```
Comment (panentheos) [inline integration-tests/README.md]: Adding these in this format makes this command no longer paste-able. Perhaps we should make sure the options are documented sufficiently above and let this be a simple example.

File: integration-tests/README.md [hunk 2]
```diff
@@ -45,10 +45,13 @@ The following environment variables are parsed by the tests:
...
 TERRA_SA_KEY=$(vault read --format=json secret/dsde/alpha/common/firecloud-account.pem | jq .data) \
 LYLE_SA_KEY=$(vault read --format=json secret/dsde/terra/envs/common/lyle-user-service-account-key | jq .data) \
 yarn test
 ```
+`HEADLESS` and `ENVIRONMENT` are optional. By default, the tests will run headless against dev.
```
Comment (panentheos) [inline integration-tests/README.md]: This appears to conflict with the default for ENVIRONMENT in the previous section.

**Review by panentheos — APPROVED** [review]
Looks good overall, see comments. Tested the code snippets and export steps.

File: integration-tests/README.md (hunk 2 above)
Comment (breilly2) [reply integration-tests/README.md]: Thanks for catching. Will fix.

File: integration-tests/README.md (hunk 1 above)
Comment (breilly2) [reply integration-tests/README.md]: Breaking copy/paste did cross my mind. I'm glad you weighed in. The full documentation of all of the options above is a bit heavy for someone looking for a convenient one-liner but could still benefit from a reminder of these two options in particular. I'll pull them out of the `sh` block and just mention them in this section.

**Pull request merged**